examples/annotated-skaffold.yaml

```diff
@@ -73,3 +73,11 @@ deploy:
     # - deployment/web-app1
     # - namespace:deployment/web-app2
     #
+
+# profiles section has all the profile information to use Google Container Builder    
```

Comment:
Profiles are not only for container builder. A profile can override any build or deploy configuration